code/modules/cargo/packs.dm

```diff
@@ -2509,12 +2509,37 @@
 
 /datum/supply_pack/misc/aquarium_fish
 	name = "Aquarium Fish Case"
-	desc = "An aquarium fish handpicked by monkeys from our collection."
+	desc = "An aquarium fish collection handpicked by monkeys from our collection."
```

Comment:
having collection in here twice feels a little awkward to me.

Reply:
changed to bundle